app/controllers/likes_controller.rb

```diff
@@ -5,7 +5,11 @@ def create
 
     respond_to do |format|
       format.html do
-        redirect_to user_post_path(post.user.id, post.id) if like.save
+        if like.save
+          redirect_to user_post_path(post.user.id, post.id)
+        else
+          flash[:alert] = 'Error: like is not published'
+        end
```

Comment:
- [ ] Kindly add flash message for successful likes

Reply:
Done✔